test/integration/sapi-util.h

```diff
@@ -97,7 +97,7 @@ create_keyedhash_key (
  * provided in the 'handle' parameter.
  * Under the covers this function uses an IV of all zeros and so it can not
  * be used for streaming. It can only be used to encrypt or decrypt a single
- * buffer.
+ * buffer. This function uses tpm to perform encrytion.
```

Comment:
s/encrytion/encryption/

Thanks for adding this.

Reply:
Fixed